src/main/webapp/WEB-INF/views/login.jsp

```diff
@@ -0,0 +1,17 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Login</title>
+    <%@include file="header.jsp" %>
```

Comment:
We don't log in already why We can logout?

Reply:
We are able to visit /login page wile already authorized. So why not?